src/common/enums/alkemio.error.status.ts

```diff
@@ -58,4 +58,7 @@ export enum AlkemioErrorStatus {
   SESSION_EXTEND = 'SESSION_EXTEND',
   BEARER_TOKEN = 'BEARER_TOKEN',
   CALLOUT_CLOSED = 'CALLOUT_CLOSED',
+  GEO_SERVICE_NOT_AVAILABLE = 'GEO_SERVICE_NOT_AVAILABLE',
+  GEO_SERVICE_ERROR = 'GEO_SERVICE_ERROR',
+  GEO_SERVICE_LIMIT_REACHED = 'GEO_SERVICE_LIMIT_REACHED',
```

Comment:
It is a standard notation to name the exception with `exceeded`. There exception occurs when you exceed the limit, not when you reach it.
```suggestion
  GEO_SERVICE_REQUEST_LIMIT_EXCEEDED = 'GEO_SERVICE_REQUEST_LIMIT_EXCEEDED',
```